src/aiida_quantumespresso/calculations/__init__.py

```diff
@@ -183,6 +183,13 @@ def validate_inputs(cls, value, port_namespace):
         if 'settings' in value:
             settings = _uppercase_dict(value['settings'].get_dict(), dict_name='settings')
 
+            if 'ADDITIONAL_RETRIEVE_LIST' in settings:
+                warnings.warn(
+                    'The key `ADDITIONAL_RETRIEVE_LIST` in the settings input is deprecated and will be removed in '
+                    'the future. Use the `CalcJob.metadata.options.additional_retrieve_list` input instead.',
+                    FutureWarning
```

Comment:
Think these warnings are not shown by default in Python. This is why we use a separate class `aiida.common.warnings.AiidaDeprecationWarning` (https://github.com/aiidateam/aiida-core/blob/f43a51010b3c84b1e9526d167401fc9ac07c556a/src/aiida/common/warnings.py#L15)

I am not sure if we were already using this approach in plugins, but may be a nice way to ensure users see the warning (and that it can be turned off through `verdi config warnings.showdeprecations False`).

Reply:
Thanks @sphuber for the review and reminder! You are right, those wouldn't be shown. I'll adapt it

Reply:
Just for context: I actually remember now why I used `FutureWarning` instead of `DeprecationWarning`, because `DeprecationWarnings` are silenced per default but `FutureWarnings` shouldn't be. Nonetheless, I think it makes sense to use the AiiDA specific subclass for the other reason outlined above.